cli/deno_error.rs

```diff
@@ -252,6 +247,8 @@ impl GetErrorKind for DlopenError {
   }
 }
 
+// NOTE(bartlomieju): seems this is necessary - can't use ErrBox here
+// TODO(bartlomieju): ultimately this should be rewritten to `RuntimeError`?
```

Comment:
> TODO: ultimately this should be rewritten to `RuntimeError`?

I think this TODO is in the wrong place. This is not a RuntimeError.

Reply:
That's right, but the downcasts are exactly for runtime errors... Anyway, still iterating on that